app/src/main/java/org/oppia/android/app/databinding/DrawableBindingAdapters.java

```diff
@@ -36,4 +36,14 @@ public static void setTopBackgroundDrawable(@NonNull View view, @ColorInt int co
     GradientDrawable background = (GradientDrawable) view.getBackground();
     background.setColor(0xff000000 | colorRgb);
   }
+
+  /** Used to set a rounded-rect background drawable with a data-bound color. */
+  @BindingAdapter("largeRoundedRectDrawableWithColor")
+  public static void setLargerBorderRadiusBackgroundDrawable(
```

Comment:
Nit: Suggesting "Large" since it's more definitive for our `24dp` case, whereas the term "Larger" is relative.
```suggestion
  public static void setLargeBorderRadiusBackgroundDrawable(
```

Reply:
updated